package.nls.json

```diff
@@ -31,5 +31,6 @@
     "azure-api-center.views.api-center-treeview.dataplane.title": "Platform API Catalog",
     "azure-api-center.commands.apiCenterWorkspace.addApis.title": "Connect to an API Center",
     "azure-api-center.commands.apiCenterWorkspace.removeApis.title": "Disconnect from API Center",
-    "azure-api-center.commands.apiCenterWorkspace.learnApiCatalog.title": "Learn about API Catalog"
+    "azure-api-center.commands.apiCenterWorkspace.learnApiCatalog.title": "Learn about API Catalog",

```

Comment:
Do you mean: "Learn about Azure API Center" or "Learn more about API Center portal"?

Reply:
@juliajuju93 In this PR, I did not change the wording of this line of code. I just add a comma symbol in this line. :)

This wording is still there with no change: 
![image](https://github.com/user-attachments/assets/64bbea61-997f-44e7-b1ad-6a3801a49870)